.prettierrc.json

```
@@ -15,13 +15,13 @@
   "plugins": ["@trivago/prettier-plugin-sort-imports"],
   "importOrder": [
     "<THIRD_PARTY_MODULES>",
-    "^types/",
-    "^hooks/",
-    "^components/|^pages/",
+    "^@/types/",
+    "^@/hooks/",
+    "^@/components/|^pages/",
```

```suggestion
    "^@/components/|^@pages/",
```

"pages" 앞에도 `@`가 붙어야 할 것 같아요

---

수정했습니다!